docs/commands/bbi_nonmem_summary.md

```diff
@@ -0,0 +1,56 @@
+## bbi nonmem summary
+
+summarize the output of model(s)
+
+### Synopsis
+
+summarize model(s), for example:
+bbi nonmem summary run001/run001
```

Comment:
> I guess I'm not sure where/how they end up being rendered for the user, so this may not matter.

Right, it depends on how the renderer handle line breaks, but 1) it probably will be and 2) either way, I think we should adjust these to look okay on GitHub.